pkg/netceptor/tlsconfig_test.go

```diff
@@ -0,0 +1,207 @@
+package netceptor
+
+import (
+	"io/ioutil"
+	"log"
+	"os"
+	"testing"
+)
+
+func setupTest(tb testing.TB) (func(tb testing.TB), os.File, os.File, os.File, os.File) {
+	log.Println("setup test")
+	/* write bad cert file to temp file */
+	badCertFile, err := ioutil.TempFile("", "*")
+	if err != nil {
+		log.Fatal(err)
+	}
+	defer os.Remove(badCertFile.Name())
+	_, err = badCertFile.Write([]byte(`
+-----BEGIN CERTIFICATE-----
```

Comment:
small detail: if there is a good way to define the certs at the top of this file, it might make this cleaner

Reply:
fixed. decided to keep the cert values as global variables in the test file.